canard.c

```diff
@@ -372,7 +372,7 @@ int16_t canardHandleRxFrame(CanardInstance* ins, const CanardCANFrame* frame, ui
             return CANARD_ERROR_OUT_OF_MEMORY;
```

Comment:
Could you please also fix this one and the one below on line 386 while you're at it?